### PR TITLE
Fixes #14264: uuid.hive is not present afet an install or an upgradie ofrudder agent to latest 4.1.x, 4.3.x and 5.0.x on rpm system purges the uuid.hive

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/14264

### CODE REVIEW

File: 12_upgrade/05_caution.txt [hunk 1]
```diff
@@ -49,6 +49,23 @@ Then, you now need to use:
 
 ====
 
+[NOTE]
+
+====
+
+Upgrading from 4.1.18 or earlier can cause transient missing /opt/rudder/etc/uuid.hive - as this
```
Comment (amousset) [inline 12_upgrade/05_caution.txt]: \`/opt/rudder/etc/uuid.hive`

File: 12_upgrade/05_caution.txt [hunk 2]
```diff
@@ -49,6 +49,23 @@ Then, you now need to use:
...
+file is no longer part of the packaging. The issue will resolve itself, and happen only once - any
+subsequent upgrade will not exhibit this problem anymore.
+
+If you don't want to wait up to 5 minutes for this issue to correct itself, you can run:
+
+    rudder agent check
```
Comment (amousset) [inline 12_upgrade/05_caution.txt]: \----
rudder agent check
\----

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
OK, squash merging this PR

**Comment by ncharles** [comment]
It's been reverted, as https://issues.rudder.io/issues/14616 fixed it

**Pull request merged**